.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -73,3 +73,14 @@ body:
         - 'No'
     validations:
       required: true
+  - type: dropdown
+    id: pr
+    attributes:
+      label: Are you willing to provide a PR?
+      description:  |
+        Providing a PR can drastically speed up the process of fixing this bug. Don't worry, it's still OK to answer 'No' :).
+      options:
+        - 'Yes'
+        - 'No'
+    validations:
+      required: true
```

Comment:
What is your logic for making this a required question?

Reply:
Was just copy pasting the value from the question above `Will you send logs?`. To me this is the same level of requirement, and I guess we want the issue submitter to answer every questions.

But happy to set this one to `false` if you prefer.